README.md

```diff
@@ -1 +1,70 @@
-# angular-routing-testing-readme
\ No newline at end of file
+# Testing Routing
+
+## Overview
+
+Now that we've got our routes setup, it's crucial that when we make changes to our application that we don't break any existing route - that's where tests come in!
+
+## Objectives
+
+- Describe routing testing
+- Write a test for routes and expect API calls/templates/Controllers
+
+## Instructions
+
+When we test our routes/states we should check that the URL is as we expect, that it's using the correct template and fetching the correct data that it needs.
```

Comment:
I'm not familiar with controller tests in Rails - is there a good article on this?